test/algorithms/buffer/test_buffer_svg_per_turn.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef BOOST_GEOMETRY_TEST_BUFFER_SVG_PER_TURN_HPP
 #define BOOST_GEOMETRY_TEST_BUFFER_SVG_PER_TURN_HPP
 
+#if defined(TEST_WITH_SVG_PER_TURN)
```

Comment:
This only needs to be compiled if this define is set.
Otherwise, `#include <boost/ptr_container/ptr_vector.hpp>` fails because `ptr_vector` is currently not in the CMakeLists and I don't want to add another dependency for this.

It's years ago I used this svg per turn, and I think it can be removed.